packages/seacas/scripts/exodus3.in.py

```diff
@@ -2345,7 +2349,7 @@ def put_elem_blk_names(self, names):
 
         >>> exo.put_elem_blk_names(elem_blk_names)
 
-        Parameters
+        Parametersf
```

Comment:
Minor: Spelling / extra f on end

Reply:
Nice catch. I will send up a different commit that has those fixes.

Reply:
I think these should be resolved now.